src/main/java/sysc4806/graduateAdmissions/controller/HomeController.java

```diff
@@ -0,0 +1,16 @@
+package sysc4806.graduateAdmissions.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@CrossOrigin(maxAge = 3600)
```

Comment:
Why is this different from default? (I don't think it needs to be changed, just curious)

Reply:
By the name, i'd assume its for [cross origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) which lets you make requests to another domain.